app/page.tsx

```diff
@@ -38,105 +44,7 @@ export default function IndexPage() {
         </div>
       </div>
 
-      <div className="flex flex-col bg-white">
-        <div className="flex flex-col justify-between gap-5 border-y border-black px-[32px] py-5 md:flex-row md:gap-0 lg:px-20">
-          <div className="flex flex-col items-start justify-center gap-2 lg:flex-row lg:items-center">
-            <div className="flex w-[280px] items-center justify-start gap-2">
-              <Image src={PSEIcon1} alt="pseicon1" height={16} width={16} />
```

Comment:
can we do something better with the alt text here, like alt="learn and share"
the icon should name it better as well but I know you are just using it as it is.

Reply:
i'm terrible on reviewing, i'm viewing the old code, my bad.

Reply:
The `alt` is meant to be read by assistive technologies like screen readers. The standard is that images that are decorative (like these icons) should have `alt=""` so its ignored by assistive technologies. I'll update accordingly!